index.js

```diff
@@ -18,6 +18,12 @@ module.exports = function(env, callback) {
       nenv.addFilter(name, filter);
     });
   }
+  if(env.config.nunjucks && env.config.nunjucks.filtermodules) {
+    env.config.nunjucks.filtermodules.map( function (name) {
+      var filter = require(name);
+      filter.install(nenv);
```

Comment:
I must be blind, but I can't find this part of the API in the [docs](https://mozilla.github.io/nunjucks/api.html). Is it new/undocumented?